SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -257,7 +300,7 @@ def customise_Reco(process,pileup):
     
     # Make pixelTracks use quadruplets
     process.pixelTracks.SeedMergerPSet = cms.PSet(
-        layerList = PixelSeedMergerQuadruplets,
+        layerListName = cms.string('PixelSeedMergerQuadruplets'),

```

Comment:
@boudoul (see the comment above) This should be kept as it was, or changed to `layerList = cms.PSet(refToPSet_ = cms.string("PixelSeedMergerQuadruplets"))`.